test/ik/test_differential_ik.py

```diff
@@ -0,0 +1,66 @@
+from os.path import dirname, join, abspath
+import numpy as np
+import pinocchio
+
+from pyroboplan.core.utils import get_random_transform
+from pyroboplan.ik.differential_ik import DifferentialIk, DifferentialIkOptions
+
+
+def load_panda_models():
```

Comment:
This is going to crop up everywhere in other tests, so I should probably make some kind of test utility... not for this PR though

Reply:
Oh yeah good call... or put it in core utils and use it in the examples too?

Reply:
I'm thinking of going as far as including some of these models in the actual package itself. Don't worry about it. for now.